pkgs/games/shipwright/default.nix

```
@@ -65,7 +62,7 @@ stdenv.mkDerivation (finalAttrs: {
   ] ++ lib.optionals stdenv.hostPlatform.isLinux [
     lsb-release
     copyDesktopItems
-  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
+  ] ++ lib.optional stdenv.hostPlatform.isDarwin [
```

`lib.optional` will wrap the 2nd arg in a list, which you don't want.
```suggestion
  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
```

---

Fixed, thank you!